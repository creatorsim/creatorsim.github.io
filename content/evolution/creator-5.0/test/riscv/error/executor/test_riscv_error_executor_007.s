
#
# Creator (https://creatorsim.github.io/creator/)
#

.data
	a: .zero 4

.text
main:
	
	la t0, a
    li t1, 10
	sw t1, 1(t0)