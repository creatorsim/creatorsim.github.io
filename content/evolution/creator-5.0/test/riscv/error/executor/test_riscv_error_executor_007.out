
The memory must be align

 Error found.
 The program has finished with errors
cr[PC]:0x10; ir[x5,t0]:0x200000; ir[x6,t1]:0xa; memory[0x200004]:0x0a; keyboard[0x0]:''; display[0x0]:''; 

